ios/firebase_app_id_file.json

```diff
@@ -0,0 +1,7 @@
+{
+  "file_generated_by": "FlutterFire CLI",
```

Comment:
Co ty na to by dodać `flutterfire_cli` jako `dev_dependencies` w `pubspec.yaml`?

Reply:
poprawie